src/libopensc/card-piv.c

```diff
@@ -160,14 +166,16 @@ typedef struct piv_private_data {
 	char * offCardCertURL;
 	int pin_preference; /* set from Discovery object */
 	int logged_in;
+	int pstate;
 	int pin_cmd_verify;
+	int context_specific;
```

Comment:
Maybe add a comment about what this field does and why. Same for other maybe not-so-obvious fields?